template.tpl

```diff
@@ -15,8 +15,8 @@ ___INFO___
   "securityGroups": [],
   "displayName": "Mixpanel",
   "brand": {
-    "id": "brand_dummy",
-    "displayName": "Custom Template",
+    "id": "github.com_stape-io",
+    "displayName": "stape-io & Mariusz Brucki",
```

Comment:
Not sure this is accept forma for gallery.
We can add this to remade and leave here as stape-io (GitHub company name)

Reply:
Sure! So you can put me somewhere there in the readme, thanks 👍

Reply:
Please add your name to readme :) It will be faster If not I will do this on weekend